packages/manager/modules/pci/src/projects/project/storages/databases/database.service.js

```diff
@@ -29,6 +38,34 @@ export default class DatabaseService {
     };
   }
 
+  activateLab(projectId, lab) {
+    return (lab.isOpen()
+      ? this.PciProjectLabsService.activateLab(projectId, lab)
+      : this.$q.resolve()
+    ).then(() =>
+      lab.isOpen() || lab.isActivationg()
```

Comment:
```suggestion
      lab.isOpen() || lab.isActivating()
```

Reply:
good catch but take care because the typo is here too : 
https://github.com/ovh/manager/blob/develop/packages/manager/modules/pci/src/components/project/labs/lab.class.js#L21

Reply:
Nice catch @marie-j . This is fixed in both the places :)